src/types/index.ts

```diff
@@ -11,6 +11,7 @@ export interface User {
 }
 
 export interface Asset {
+  // TODO: add vToken
```

Comment:
If we did this I think we could reduce passing tokens and vToken as props in repay and borrow components

Reply:
We'd still pass them individually to improve modularity, but yes this will be done as the refactors are happening